src/Microsoft.Health.Blob/Features/Storage/BlobClientProvider.cs

```diff
@@ -42,7 +42,7 @@ public class BlobClientProvider : IHostedService, IRequireInitializationOnFirstR
         public Task StartAsync(CancellationToken cancellationToken)
         {
             // The result is ignored and will be awaited in EnsureInitialized(). Exceptions are logged within DocumentClientInitializer.
-            _ = _initializationOperation.EnsureInitialized();
+            _ = _initializationOperation.EnsureInitialized().AsTask();
```

Comment:
`ValueTask` is not meant to run in this manner; it is meant to be immediately awaited and may cause issues with invoking `AsTask()`